modeling/meters.md

```diff
@@ -0,0 +1,132 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+Meters
+======
+
+```{note}
+This feature is new in Brick v1.3
+```
+
+Brick provides a basic but flexible model for describing meters, submeter hierarchies, and their relationships to the building.
+Meters are equipment that measure the consumption or production of energy, steam, gas or other "substances" in the building.
+
+
+## Meters and Submeters
+
+Meters are instances of the `brick:Meter` class or any of its subclasses.
+Submeter hierarchies are defined through the `brick:hasSubMeter` and `brick:isSubMeterOf` relationships.
+In fact, meters can *only* be related to each other via these relationships.
+
+```ttl
+:building_meter a brick:Building_Meter ;
+    brick:hasSubMeter :floor-meter-1, :floor-meter-2 .
+:floor-meter-1 a brick:Meter .
+:floor-meter-2 a brick:Meter .
+```
+
+```{warning}
+Brick does not add any properties to the submeter relationship --- it only indicates that there is one. If this is a problem, [let us know!](https://github.com/BrickSchema/Brick/issues)
```

Comment:
Both meters and submeters may have `brick:meters` relationships as described below. I think saying that exists here could avoid confusion in the future.

Reply:
"submeter" is a relationship, not an entity type. I tried to clarify this in the new text